servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -552,4 +602,52 @@ public Completable closeAsyncGracefully() {
             return closeable.closeAsyncGracefully();
         }
     }
+
+    private static final class SdStatusCompletable extends Completable implements CompletableSource.Processor {
```

Comment:
Do we need the complete `Processor` contract here? 

If we are just using `onComplet()` and `onError()` then not using the `Processor` contract helps remove the extra `subscribe()` method and delegation to `subscribeInternal()`